default/stringtables/de.txt

```diff
@@ -1115,7 +1115,7 @@ OPTIONS_DB_QUICKSTART
 Beginnt ein neues Schnellstartspiel. Das Hauptmenü wird umgangen.
 
 OPTIONS_DB_CHECKED_GL_VERSION
-Speichert ob die OpenGL Version des Systems überprüft wurde. Wenn nicht, werden einige Zeichenoptionen abhängig von der GL Version angepasst, nachdem die Version überprüft wurde.
+Speichert ob die OpenGL Version des Systems nicht überprüft wurde. Wenn falsche, werden einige Zeichenoptionen abhängig von der GL Version angepasst, nachdem die Version überprüft wurde.
```

Comment:
Native speaker reviews requested on this commit for spanish, french, german, and russian.  The logic swapped but some of these do not seem to translate to show such logic to begin with.

If any lack review, I would opt to remove that particular translation than to have potentially incorrect description.